src/EventSubscriber/PostRowSave.php

```diff
@@ -56,9 +68,42 @@ public static function getSubscribedEvents() {
    * @throws \Drupal\migrate\MigrateSkipRowException
    */
   public function onMigratePostRowSave(MigratePostRowSaveEvent $event) {
-    // $migration = $event->getMigration();
-    // $row = $event->getRow();
-    // $nids = $event->getDestinationIdValues();
+    $row = $event->getRow();
+    $migration_tools_settings = $row->getSourceProperty('migration_tools');
+
+    if (!empty($migration_tools_settings)) {
+      // Create redirects if enabled.
+      // @todo Current only supports 1st migration_tools array entry.
+      $source_type = $migration_tools_settings[0]['source_type'];
```

Comment:
Is there any chance this won't exist?   Seems like a deep reach for something that has not been looked for first.